src/tree/v2/workspace/registerWorkspaceTreeV2.ts

```diff
@@ -0,0 +1,34 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { WorkspaceResourceProviderManager } from '../../../api/v2/WorkspaceResourceProviderManager';
+import { WorkspaceResourceBranchDataProviderManager } from './WorkspaceResourceBranchDataProviderManager';
+import { WorkspaceTreeDataProvider } from './WorkspaceTreeDataProvider';
+import { localize } from './../../../utils/localize';
+
+export function registerWorkspaceTreeV2(
+    branchDataProviderManager: WorkspaceResourceBranchDataProviderManager,
+    context: vscode.ExtensionContext,
+    refreshEvent: vscode.Event<void>,
+    workspaceResourceProviderManager: WorkspaceResourceProviderManager): void {
+    const treeDataProvider = new WorkspaceTreeDataProvider(
+        branchDataProviderManager,
+        refreshEvent,
+        workspaceResourceProviderManager);
+
+    context.subscriptions.push(treeDataProvider);
+
+    const treeView = vscode.window.createTreeView(
+        'azureWorkspaceV2',
+        {
+            canSelectMany: true,
```

Comment:
```suggestion
            canSelectMany: true,
            showCollapseAll: true,
```

Reply:
I'm ok with that.  (I was just copying the existing V1 configuration, which omits collapse-all for the workspaces view, assuming it was a specific design choice.)

Reply:
Updated.